gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/service/FhirIDGeneratorService.java

```diff
@@ -0,0 +1,15 @@
+package uk.nhs.adaptors.pss.translator.service;
+
+import java.util.UUID;
+
+import org.springframework.stereotype.Service;
+
+@Service
+public class FhirIDGeneratorService {
```

Comment:
Fhir**Id**GeneratorService will be better, it will be consistent with other class names (all use PascalCase)